sdk/documenttranslation/azure-ai-documenttranslation/azure/ai/documenttranslation/_client.py

```diff
@@ -75,12 +88,28 @@ def close(self):
     @distributed_trace
     def create_translation_job(self, inputs, **kwargs):
         # type: (List[DocumentTranslationInput], **Any) -> JobStatusResult
-        """
+        """Create a document translation job which translates the document(s) in your source container
+        to your target container in the given language.
+
+        For supported languages and document formats, see the service documentation:
+        https://docs.microsoft.com/azure/cognitive-services/translator/document-translation/overview
 
-        :param inputs:
+        :param inputs: A list of translation inputs. Each individual input has a single
+            source URL to documents and can contain multiple target URLs (one for each language)
```

Comment:
idk how important it could be, but this makes me think that `target` is only a list of URLs. Maybe saying something like `and can contain multiple TargetInput` or whatever the name of the object